wolfcrypt/src/wc_port.c

```diff
@@ -441,7 +441,11 @@ int wc_ReadDirFirst(ReadDirCtx* ctx, const char* path, char** name)
     XSTRNCPY(ctx->name, path, MAX_FILENAME_SZ - 3);
     XSTRNCPY(ctx->name + pathLen, "\\*", MAX_FILENAME_SZ - pathLen);
 
+    #ifdef _WIN32_WCE
+    ctx->hFind = FindFirstFileA((LPCWSTR) ctx->name, &ctx->FindFileData);
```

Comment:
wc_port.h has a macro for `FindFirstFileA ` and `_WIN32_WCE`. Seems like these differences should be solved in the macro...?

`#define FindFirstFileA(fn, d) FindFirstFile(fn, (LPWIN32_FIND_DATAW) d)`

Reply:
Fantastic suggestion. Done!